synth_node_bin/src/action/send_get_addr_and_forever_sleep.rs

```diff
@@ -0,0 +1,36 @@
+use std::net::SocketAddr;
+
+use anyhow::Result;
+use tokio::time::{sleep, Duration};
+use ziggurat_zcash::{protocol::message::Message, tools::synthetic_node::SyntheticNode};
+
+use super::SynthNodeAction;
+
+pub struct Action;
+
+#[async_trait::async_trait]
+impl SynthNodeAction for Action {
+    fn info(&self) -> &str {
+        "request an GetAddr and then sleeps forever"
+    }
+
+    #[allow(unused_variables)]
+    async fn run(&self, synth_node: &mut SyntheticNode, addr: SocketAddr) -> Result<()> {
+        println!("Synthetic node performs an action.");
```

Comment:
Maybe we should focus on one type of logging/printing. If we're using `tracing::` so maybe we'd convert that to `tracing::info!`

Reply:
I think the binary should have a stdout printout as well in case the RUST_LOG is not used.

So the most important info - like action info, was the synt node started/restarted could be printed with `println`.
Because I don't always start it with `RUST_LOG`

Reply:
Convinced me 😸